modules/pricer/src/main/java/com/opengamma/strata/pricer/impl/swap/DiscountingKnownAmountPaymentPeriodPricer.java

```diff
@@ -64,6 +64,11 @@ public double forecastValue(KnownAmountPaymentPeriod period, RatesProvider provi
   }
 
   @Override
+  public double presentValueCashFlowEquivalent(KnownAmountPaymentPeriod period, RatesProvider provider) {
```

Comment:
Why is there the need of a "presentValueCashFlowEquivalent"; the existing presentValue should the sufficient.